### PR TITLE
fix `Unable to test Capacitor.js application` (close #2094)

### LINKED ISSUE #2094: Unable to test Capacitor.js application
### What is your Test Scenario?
I have ported cordova app to capacitor, but it seems testcafe proxy has problem with capacitor plugins.
 
### What is the Current behavior?
Right after page is loaded I can see errors in console. 
```
ERROR Error: Uncaught (in promise): hammerhead|processed-context does not have web implementation.
```
This type of message originates from Capacitor Plugins API. It happens in case a plugin is used on web in case it has not web implementation, which is not true in this case.
 
### What is the Expected behavior?
Testcafe should handle proxying of capacitor plugins correctly.
 
### What is your web application and your TestCafe test code?
I created github repo with sample app.
 
https://github.com/martinkasa/testcafe-capacitor

### Steps to Reproduce:

1. git clone https://github.com/martinkasa/testcafe-capacitor.git
2. cd testcafe-capacitor
3. npm install
4. ionic serve
5. wait until app is started and leave it running
6. npx testcafe "chrome" ./testcafe.ts

When browser is opened, check console with mentioned errors.
 
### Your Environment details:
 
* testcafe version:                   1.3.3
* node.js version:                    v12.3.0
* browser name and version: Chrome 75.0.3770 / Mac OS X 10.14.6

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

File: src/client/sandbox/node/window.ts [hunk 1]
```diff
@@ -299,6 +298,10 @@ export default class WindowSandbox extends SandboxBase {
         /*eslint-enable no-restricted-properties*/
     }
 
+    isProxyObject (obj: any): boolean {
```
Comment (miherlosev) [inline src/client/sandbox/node/window.ts]: `static isProxyObject(obj: any)`

File: src/client/sandbox/node/window.ts [hunk 2]
```diff
@@ -76,6 +76,9 @@ const SANDBOX_DOM_TOKEN_LIST           = 'hammerhead|sandbox-dom-token-list';
 const SANDBOX_DOM_TOKEN_LIST_OWNER     = 'hammerhead|sandbox-dom-token-list-owner';
 const SANDBOX_DOM_TOKEN_LIST_UPDATE_FN = 'hammerhead|sandbox-dom-token-list-update';
 
+const IS_PROXY_OBJECT        = 'hammerhead|is-proxy-object';
```
Comment (miherlosev) [inline src/client/sandbox/node/window.ts]: const IS_PROXY_OBJECT_INTERNAL_PROP_NAME  = 'hammerhead|is-proxy-object|internal-prop-name';
const IS_PROXY_OBJECT_INTERNAL_PROP_VALUE  = 'hammerhead|is-proxy-object|internal-prop-value';

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Pull request merged**